src/proxy.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+/**
+ * @class Proxy
+ * @classdesc Proxy object that hides runtime details (browser or node)
+ *  from the implementation. Is configured in the entry points shark-browser.js or shark-node.js.
+ */
+module.exports = {}
```

Comment:
Is there a better name for this file, since [Proxy](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Proxy) already exists in Javascript?

Reply:
`env`?

Reply:
`env`?

Reply:
I don't think that's really an improvement. 
Actually proxy is the correct name, but I could use it like
```
const SharkProxy = require('/src/proxy.js')
return SharkProxy.fetch(...)
```

Reply:
`SharkProxy` sounds OK